s4e-web/src/app/state/session/session.service.ts

```diff
@@ -60,8 +64,19 @@ export class SessionService {
 
   @action('logout')
   logout() {
-    this.setToken(null, null);
-    resetStores();
-    this.router.navigate(['/login']);
+    if (this.sessionQuery.isLoggedIn()) {
+      console.log("JESTEM");
+      this.setToken(null, null);
+
+      const stores = persistState();
+      stores.clearStore('Notification');
+
+      akitaConfig({
+        resettable: true
+      });
```

Comment:
Why is this needed? Shouldn't config be done in some initialization code, and not in logout function?